addons/bus/models/ir_async.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+from odoo import models, fields
+
+_logger = logging.getLogger(__name__)
+
+
+class IrAsync(models.Model):
+    _inherit = "ir.async"
+
+    notify = fields.Boolean(default=False)
+
+    def call_notify(self, description, method, *args, **kwargs):
```

Comment:
why is this public?

Reply:
Because it needs a bound method to work and that I wouldn't like to write `env['ir.async']._call` everywhere. The function can de-facto only be used within python code and not directly via rpc. We can discuss whether we want to force the method to start with a `_` prefix.

Reply:
Nothing guarantees that this method will never have side-effects, so yes, we need it to be marked private so that it can *never* be invoked via RPC, no matter what. It's as simple as that. Cosmetics are irrelevant.

Reply:
It's made wore because most of the code downstack uses sudo() or methods that do not have any ACL control at all (raw SQL, bus notifs, etc.)

Reply:
made private